models/event.py

```diff
@@ -227,6 +227,14 @@ def week(self):
 
         return self._week
 
+    @property
+    def week_str(self):
```

Comment:
Can we write a quick test for this? Should be straightforward

Reply:
Opened #2401 to deal with this.